news/2023/2023-09-20-project-loved-six-years-september-2023.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: post
+title: "Project Loved: Six Years / September 2023"
+date: 2023-09-20 22:00:00 +0000
+---
+
+6 years on, Project Loved is going strong!
+
+![](/wiki/shared/news/banners/project-loved-2.jpg)
+
+Welcome or welcome back to another edition of Project Loved! For those who are not yet familiar with who we are and what we do, Project Loved hosts community voting polls each month to decide which beatmaps get added to the [Loved category](https://osu.ppy.sh/beatmapsets?s=loved). All of the maps are chosen by the [Loved captains](https://loved.sh/team) of their respective game modes. Each beatmap banner in this news post will redirect you to the respective poll, or you can view them all directly on the [Project Loved forum](https://osu.ppy.sh/community/forums/120). The voting process lasts for 10 days, so don't forget to vote! More info can be found on the [wiki page](/wiki/Community/Project_Loved) and [website](https://loved.sh).
+
+As we usually do, we would like to thank everyone for your participation in the August votes. If you would like to view the results from the previous round, you may do so by clicking the following links:
+
+- [Results for osu!](https://osu.ppy.sh/community/forums/posts/9254939)
+- [Results for osu!taiko](ahttps://osu.ppy.sh/community/forums/posts/9254938)
```

Comment:
```suggestion
- [Results for osu!taiko](https://osu.ppy.sh/community/forums/posts/9254938)
```

Reply:
fixed on website, it'll be applied here next time Hivie updates.